### PR TITLE
Project pagination

### PR DESCRIPTION
**Pull request by julienmachon**
Fixes BlueBrain/nexus#433

I recommend going commit by commit. It looks like a lot of changes but it's mostly noise.

### CODE REVIEW

File: src/shared/store/actions/nexus/index.ts [hunk 1]
```diff
@@ -1,318 +1,43 @@
...
-      const org: Organization = await nexus.getOrganization(orgName);
-      return dispatch(fetchOrgSuccessAction(org));
-    } catch (e) {
-      return dispatch(fetchOrgFailureAction(e));
-    }
-  };
-};
 
 export const fetchProjects: ActionCreator<ThunkAction> = (name: string) => {
+  console.log('plop');
```
Comment (mdupont) [inline src/shared/store/actions/nexus/index.ts]: plop

**Comment by julienmachon** [comment]
Now the entire file is actually gone... :dash:

File: src/shared/store/actions/nexus/index.ts (hunk 1 above)
Comment (julienmachon) [reply src/shared/store/actions/nexus/index.ts]: this too, is gone :dash:

File: src/shared/components/Animations/AnimatedList.tsx [hunk 2]
```diff
@@ -40,7 +40,8 @@ const AnimatedList: React.FunctionComponent<AnimatedListProps<any>> = props => {
   if (paginationSettings) {
     const { from, total, pageSize } = paginationSettings;
     const totalPages = Math.ceil(total / pageSize);
-    const currentPage = Math.ceil(from / pageSize);
+    const currentPage = Math.round(from / pageSize) + 1;
+    console.log(currentPage);
```
Comment (mdupont) [inline src/shared/components/Animations/AnimatedList.tsx]: clog

Comment (julienmachon) [reply src/shared/components/Animations/AnimatedList.tsx]: gone

**Pull request merged**